### PR TITLE
1.5.x fast

### PR DESCRIPTION
**Pull request by brian-armstrong-discord**
distillery prep + fast_start

### CODE REVIEW

File: priv/templates/fast_boot.eex [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+set -o posix
+set -e
+
+unset CDPATH
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd -P)"
+RELEASE_ROOT_DIR="$(cd "$SCRIPT_DIR/.." && pwd)"
+RELEASES_DIR="$RELEASE_ROOT_DIR/releases"
+REL_NAME="${REL_NAME:-discord_voice}"
```
Comment (birarda) [inline priv/templates/fast_boot.eex]: discord_voice?

**Review by birarda — CHANGES_REQUESTED** [review]

File: priv/templates/fast_boot.eex (hunk 1 above)
Comment (brian-armstrong-discord) [reply priv/templates/fast_boot.eex]: woops

File: priv/libexec/commands/prep.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -o posix
+
+## This command starts the release in the foreground, i.e.
+## standard out is routed to the current terminal session.
```
Comment (birarda) [inline priv/libexec/commands/prep.sh]: Guessing this isn't a valid comment anymore

File: priv/libexec/commands/prep.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    -env ERL_LIBS "$REL_LIB_DIR" \
+    -pa "$CONSOLIDATED_DIR" \
+    -args_file "$VMARGS_PATH" \
+    -config "$SYS_CONFIG_PATH" \
+    -mode "$CODE_LOADING_MODE" \
+    ${ERL_OPTS} \
+    -extra ${EXTRA_OPTS}
+
+IFS=$'\n' CURENV=($(printenv | grep -v STARTENV | sed -e 's/=/="/' | sed -e 's/$/"/'))
+IFS='#' read -r -a startenv_arr <<< "$STARTENV"
```
Comment (brian-armstrong-discord) [inline priv/libexec/commands/prep.sh]: one tricky thing about this is that we rely on a `#` separator for all the env. we need a character that won't appear anywhere in any env var. not my favorite part of this

File: priv/templates/boot_check.eex [hunk 4]
```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
+curenv=($(printenv | sed -e 's/=/="/' | sed -e 's/$/"/'))
```
Comment (brian-armstrong-discord) [inline priv/templates/boot_check.eex]: actually I think if we leave this as a plainstring and don't array-ize it, we can just rely on \n separator safely

**Review by birarda — APPROVED** [review]

**Pull request merged**